webapp/src/pages/offer/components/OfferComponent.js

```diff
@@ -0,0 +1,202 @@
+import React, {Component} from "react";
+import {withRouter} from "react-router";
+import { offerServices  } from "../../../services/offer/offer.service";
+import { profileService} from "../../../services/profilelist/profilelist.service";
+
+const DEFAULT_EVENT_ID = process.env.REACT_APP_DEFAULT_EVENT_ID || 1;
+
+class Offer extends Component {
+  
+  constructor(props) {
+        super(props);
+  
+        this.state = {
+          user:{},
+          userProfile : [],
+          loading: true,
+          error: "",
+          offer_id: null,
+          rejected_reason:"",
+          rejected:false,
+          showReasonBox:false,
+          accepted: false,
+          offerList: [],
+          category:""
+        };
+      }
+    
+      
+      handleChange = field => {
```

Comment:
is this even used? I don't think so

Reply:
Yes, during takes takes the value passed when entering rejection reason. event handler.